### DISCUSSION TITLE
Python API to capture images or video from GoPro Hero 12 via USB.

### DISCUSSION THREAD

**Discussion by AthenasArch** [gopro/labs#881] [Q&A]
Hey guys,

I'm working on a project that involves using a GoPro as a camera to capture images, which will be analyzed by an artificial intelligence system. I need the camera to be activated and, through a programming language, I can transfer the captured images to an operating system. Later, these images will be used in image analysis software.

I would like to know if anyone has managed to integrate the GoPro camera with some high-quality video API to capture images or videos directly to a PC via USB. If so, how was it done? It is crucial that communication is persistent and carried out via USB to avoid failures.

**Comment by dnewman-gpsw** [comment]
Sounds like a question for the Open GoPro team https://gopro.com/en/us/info/open-gopro
